resources/definitions/fdmprinter.def.json

```diff
@@ -6954,6 +6954,203 @@
                     "type": "float",
                     "enabled": "bridge_settings_enabled and bridge_enable_more_layers",
                     "settable_per_mesh": true
+                },
+                "clean_between_layers": 
```

Comment:
Another thing: If you want these to show up not just when specifically searching for it, but also browsing the settings, you need to put them in one of the `.cfg` files in `resources/setting_visibility`. I'd recommend putting all of them in 'expert' though.

Reply:
Thank you! All of them were put into exert.cfg.